src/main/docs/guide/observation.adoc

```diff
@@ -0,0 +1,23 @@
+== Micrometer Observation
+
+The Micronaut Micrometer Observation module simplifies the process of instrumenting your code for gathering traces and metrics. To get started, add the following dependency to your project:
+
+dependency:micronaut-micrometer-observation[groupId="io.micronaut.micrometer", scope="implementation"]
+
+This module enables you to leverage the https://micrometer.io/docs/observation#_using_annotations_with_observed[`@Observed`] annotation in your code, making it easier to monitor your application. Additionally, it allows you to create custom Observations.
+
+When your application registers a singleton instance of the `MeterRegistry` class, the Observation API will export metrics. Similarly, registering a singleton instance of the `Trace` class will enable the export of traces.
```

Comment:
what should users do? Register a singleton of MeterRegistry or just inject a bean of type MeterRegistry in their programs and it will be initliazed? same for Trace